service/gateway/metadata_handler.go

```diff
@@ -124,3 +124,124 @@ func (gateway *Gateway) listObjectsByBucketNameHandler(w http.ResponseWriter, r
 	w.Header().Set(model.ContentTypeHeader, model.ContentTypeJSONHeaderValue)
 	w.Write(b.Bytes())
 }
+
+// getObjectMetaByNameHandler handle get object info by name request
+func (gateway *Gateway) getObjectByNameHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Rename `getObjectByNameHandler` to `getObjectMetaByNameHandler`?

Reply:
done